guides/source/form_helpers.md

```diff
@@ -64,184 +64,227 @@ To create this form you will use `form_with` and the form builder object it yiel
 
 #### Checkboxes
 
-Checkboxes are form controls that give the user a set of options they can enable or disable:
+A Checkbox is a form control that allows for a single value to be selected or deselected. A group of Checkboxes are generally used to allow a user a choose one or more options from the group.
+
+For example, these two checkboxes in a form:
 
 ```erb
-<%= form.check_box :pet_dog %>
-<%= form.label :pet_dog, "I own a dog" %>
-<%= form.check_box :pet_cat %>
-<%= form.label :pet_cat, "I own a cat" %>
+<%= form.check_box :biography %>
+<%= form.label :biography, "Biography" %>
+<%= form.check_box :romance %>
+<%= form.label :romance, "Romance" %>
+<%= form.check_box :mystery %>
+<%= form.label :mystery, "Mystery" %>
 ```
 
-This generates the following:
+Will generate the following:
 
 ```html
-<input type="checkbox" id="pet_dog" name="pet_dog" value="1" />
-<label for="pet_dog">I own a dog</label>
-<input type="checkbox" id="pet_cat" name="pet_cat" value="1" />
-<label for="pet_cat">I own a cat</label>
+<input name="biography" type="hidden" value="0" autocomplete="off"><input type="checkbox" value="1" name="biography" id="biography">
+<label for="biography">Biography</label>
+<input name="romance" type="hidden" value="0" autocomplete="off"><input type="checkbox" value="1" name="romance" id="romance">
+<label for="romance">Romance</label>
+<input name="mystery" type="hidden" value="0" autocomplete="off"><input type="checkbox" value="1" name="mystery" id="mystery">
+<label for="mystery">Mystery</label>
 ```
 
-The first parameter to [`check_box`](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-check_box) is the name of the input. The checkbox's values (the values that will appear in `params`) can optionally be specified using the third and fourth parameters. See the API documentation for details.
+The first parameter to [`check_box`](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-check_box) is the name of the input which can be found in the `params` hash. If the user has checked the "Biography" checkbox only, the `params` hash would contain `params: {"biography"=>"1", "romance"=>"0", "mystery"=>"0"}`. You can use `params[:biography]` to check if that checkbox is checked by the user.
+
+The checkbox's values (the values that will appear in `params`) can optionally be specified using the third and fourth parameters. See the [API documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/FormHelper.html#method-i-check_box) for more details.
 
 #### Radio Buttons
 
-Radio buttons, while similar to checkboxes, are controls that specify a set of options in which they are mutually exclusive (i.e., the user can only pick one):
+Radio buttons are form controls that allow the user to select one option from a set of options. The user can only select one option at a time as the choices are mutually exclusive.
```

Comment:
"one option" is repeated here twice. Do we need "mutually exclusive" here? Can we combine these sentences into one?
"Radio buttons are form controls that only allow the user to select one option at a time from the list of choices."